File: src/teams/IdleTeam.scala
package teams
import battlecode.common.{Clock, GameActionException}

/**
  * Team that just keeps the archons idle.
  */
class IdleTeam extends RobotTeam {
  @throws(classOf[GameActionException])
  override def run(): Unit =
    while (true)
      Clock.`yield`()
}

File: src/teams/IdleTeam.java
package teams;

import battlecode.common.Clock;
import battlecode.common.GameActionException;

/**
  * Team that just keeps the Archons idle.
  */
public class IdleTeam implements RobotTeam {

    @Override
    public void run() throws GameActionException {
        while (true)
            Clock.yield();
    }
}
